lib/Zonemaster/GUI.pm

```diff
@@ -6,4 +6,23 @@ use strict;
 use warnings;
 use 5.14.2;
 
+=head1 NAME
+
+Zonemaster::GUI - Web GUI for Zonemaster Backend
```

Comment:
I feel "Web GUI for Zonemaster" is more correct. Backend is just a layer to reach Engine.

Reply:
Both wordings are equally correct. The difference is that the Backend-wording is more specific.

I don't agree that Backend is just a layer to reach Engine. Backend is significantly more than that. It also has a records keeping database which is a very important feature. This records keeping functionality is exposed through the GUI.

Reply:
I agree that backend adds functions. The wording is of less importance.